docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
I think for the production usecase (`flask db upgrade`) we could even go as far as recommending people to use a custom command (#2163). However, while that deals with the app chances are good that a well-written application uses an environment variable to load its config file.

Not sure if that's within Flask's scope though - adding an `export MYAPP_CFG=/path/to/myapp.cfg` to the `.profile` on the production machine might be the cleaner solution for that.